### PR TITLE
[KYUUBI #1620] Implement api: /${version}/operations/${operation_identifier}/log

### LINKED ISSUE #1620: [SUB-TASK][KPIP-1] Implement api: /${version}/operations/${operation_identifier}/log
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues?q=is%3Aissue) and found no similar issues.

### Describe the feature

Implement api: /${version}/operations/${operation_identifier}/log

### Motivation

_No response_

### Describe the solution

_No response_

### Additional context

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by simon824**
### _Why are the changes needed?_
closes #1620
 Implement api: /${version}/operations/${operation_identifier}/log

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala [hunk 1]
```diff
@@ -121,4 +121,30 @@ private[v1] class OperationsResource extends ApiRequestContext {
...
+      "get operation log")
+  @GET
+  @Path("{operationHandle}/log")
+  def getOperationLog(
+      @PathParam("operationHandle") operationHandleStr: String,
+      @QueryParam("maxrows") maxRows: Int): OperationLog = {
+    try {
+      val rowSet = backendService.sessionManager.operationManager.getOperationLogRowSet(
+        parseOperationHandle(operationHandleStr),
+        FetchOrientation.FETCH_LAST,
```
Comment (yaooqinn) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala:139]: FETCH_NEXT？

File: kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala [hunk 2]
```diff
@@ -121,4 +121,30 @@ private[v1] class OperationsResource extends ApiRequestContext {
...
+    try {
+      val rowSet = backendService.sessionManager.operationManager.getOperationLogRowSet(
+        parseOperationHandle(operationHandleStr),
+        FetchOrientation.FETCH_NEXT,
+        maxRows)
+      val logRowSet = rowSet.getColumns.get(0).getStringVal.getValues.asScala
+      OperationLog(logRowSet, logRowSet.size)
+    } catch {
+      case NonFatal(e) =>
+        e.printStackTrace()
```
Comment (yaooqinn) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/OperationsResource.scala:145]: any special reason to add this？

**Review by yaooqinn — APPROVED** [review]

**Review by yanghua — APPROVED** [review]
+1

**Comment by yaooqinn** [comment]
thanks, merged to master

**Pull request closed without merging**